### DISCUSSION TITLE
Wo lege ich zusätzliche js und css libraries am Server ab?

### DISCUSSION THREAD

**Discussion by ue-black** [e-netze/datalinq-community#20] [Q&A] [answered]
Hallo!

Kann mir jemand sagen wo ich die zusätzlichen js libraries für den Klimapass am Server ablegen muss damit Datalinq diese auch findet?

Hier?
..\DotNet.Core\api\app_5_24_2801\wwwroot\_content\DataLinq.Web\lib

Oder im Portal?

Danke für die Hinweise! LG Klaus

**Comment by gis-eni** [comment, marked as answer]
Grundsätzlich ist es egal, wo die Libraries ablegst. Sie müssen nur für den Client (Browser) erreichbar sein. Du kannst sie natürlich auch irgendwo in WebGIS API oder Portal "content/..../..." Verzeichnis legen. Da musst du aber immer aufpassen, dass sie beim nächsten Update wieder mit kopiert werden.
Besser ist es, wenn du dir auf einem Server (unabhängig von WebGIS) ein virtuelles Verzeichnis im IIS anlegst, wo du diese Scripts ablegst. So eine Art interner CDN. Dort kann dann jeder zugreifen und du musst bei Updates nicht aufpassen.

**Comment by ue-black** [comment]
Danke für die schnelle Antwort.
